rocca/agents/core.py

```diff
@@ -150,7 +163,9 @@ def pln_bc(self, query, vardecl=None, maxiter=10, rules=[]):
 
         """
 
+        # TODO: bring back multiprocessing
         agent_log.fine("pln_bc(query={}, maxiter={})".format(query, maxiter))
+        logger.info("pln_bc(query={}, maxiter={})".format(query, maxiter))
```

Comment:
It would be important to understand what you find cumbersome.  I tried to make parameterizing and using logs as easy as possible.  Here are example of how to modify their attributes (but you must know that already):

https://github.com/opencog/rocca/blob/dfd4670f6272be1819d634f9d7f9a5f548beddaa/examples/cartpole.py#L302-L308

https://github.com/opencog/rocca/blob/dfd4670f6272be1819d634f9d7f9a5f548beddaa/examples/chase.py#L111-L120

Also, in case you didn't know, you can also have each log point to its own file, etc.

How would you change the logging system so that you don't need these duplicates?